docs/src/main/sphinx/connector/firebolt.rst

```diff
@@ -0,0 +1,263 @@
+====================
```

Comment:
Please convert the whole doc to md - you can do so with rst2myst  .. see https://pypi.org/project/rst-to-myst/